### PR TITLE
Batch script render

### PR DESCRIPTION
**Pull request by jburel**
Batch script for workshop
cc @pwalczysko

### CODE REVIEW

File: maintenance/scripts/apply_rnd_settings.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	imageids=`$omero hql --ids-only --limit 500 --style csv -q "select img from DatasetImageLink l join l.parent as ds join l.child as img where ds.name = '$f1'"`
+	IFS=',' read -r -a array <<< $imageids
+	
+	for imageid in "${array[@]}"
+	do
+		imageid=${imageid/ */}
+		if [[ $imageid == Image* ]]
+		then
+			printf 'Applying rendering settings %s (dataset %s) to %s \n' "$f2" "$f1" "$imageid"
+			$omero render edit $imageid $f2
```
Comment (pwalczysko) [inline maintenance/scripts/apply_rnd_settings.sh]: ``edit`` -> ``set`` (Dom renamed ``edit`` to ``set``.  ``edit`` is still available, but deprecated.

**Comment by jburel** [comment]
@pwalczysko I have pushed this version on outreach and tested 
everything seems to work

File: maintenance/scripts/apply_rnd_settings_as.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# The render plugin currently doesn't support applying a set of rendering settings to a 
+# whole dataset. This script just itereates over a tab separated mapping file which
+# maps a dataset name to a renderings settings file (can be json or yml).
+# It retrieves the images ids of the datasets and calls the render plugin for each
+# of the images. 
+#
+# Note: 
+# - You need to log in first before running this script.
```
Comment (pwalczysko) [inline maintenance/scripts/apply_rnd_settings_as.sh]: Remove this line. This is not true anymore.

**Comment by pwalczysko** [comment]
Yes, tested myself as well, the script works fine now, thank you, ready to merge.

**Comment by jburel** [comment]
Line removed and fixed typo
Merging

**Pull request merged**